ndb/src/google/cloud/ndb/model.py

```diff
@@ -1748,8 +1754,8 @@ class BlobProperty(Property):
     def __init__(
         self,
         name=None,
-        compressed=None,
         *,
+        compressed=None,
```

Comment:
I realized when reading the `DateTimeProperty` source that `compressed` was supposed to be positional only

Reply:
I'm not sure I understand the comment. Here, you've made it keyword only.

Reply:
Yes, I wrote that in a hurry. I meant to say "was supposed to be keyword only".

Reply:
:+1: